src/Protocol/DFRN.php

```diff
@@ -1831,7 +1913,8 @@ private static function process_suggestion($xpath, $suggestion, $importer) {
 			intval(0)
 		);
 
-		notification(array(
+		notification(
```

Comment:
I don’t see a problem with the former indentation, please revert it.

Reply:
Another suggestion from the tool, supposedly PRS2 something about the opening paren being the last symbol on the line. I will change it back